data/filter.go

```diff
@@ -30,6 +30,27 @@ type ContentType struct {
 	SubTypes        []string `json:"sub_types"`
 }
 
+var defaultContentTypes = "article," +
```

Comment:
this would be cleaner as a string literal

Reply:
done like this so it was easier to add/remove/maintain the list going forward and so it was consistent with search-api https://github.com/ONSdigital/dp-search-api/blob/develop/api/search.go#L17